### DISCUSSION TITLE
Why are only some PLPROC functions included?

### DISCUSSION THREAD

**Discussion by RyanConway91** [pypest/pypestutils#26] [General]
It seems like alluv_boundary_interp in not included in the wrapped functions. Is this intentional? Why are some PLPROC functions included and others are not?

**Comment by jtwhite79** [comment]
pypestutils is not a wrapper around plproc.  We only have the "greatest hits" from plproc (and the pest utils in general) coded up in the fortran library

**Comment by RyanConway91** [comment]
Understood, thanks. 

Out of curiosity, do practitioners often use spatially variable bearing/anisotropy? Is there a better way to come up with these in alluvial environments than alluv_boundary_interp?

Thanks

**Comment by jtwhite79** [comment]
I think the spatially varying hyperpars is pretty new...but it can do some pretty impressive things re higher-order geostat fields, including (trying to) approximate expected heterogeneity from alluvial processes.  And so far, Ive found that the ensemble gradient approximation can accommodate the added nonlinearity that these hyperpars bring during history matching...but its early days still and Im sure there are situations where the nonlinearity will be a problem. FWIW @briochh and I have some some stuff in-progress that uses these hyperpars to make fields like this (and we can match water levels to make similar fields):
![image](https://github.com/pypest/pypestutils/assets/1770912/0d5a8c0f-356d-4a8f-ac1a-98f634e185f0)
